epochStart/bootstrap/syncValidatorStatus.go

```diff
@@ -56,13 +62,51 @@ func NewSyncValidatorStatus(args ArgsNewSyncValidatorStatus) (*syncValidatorStat
 		return nil, err
 	}
 
-	argsNodesCoordinator := ArgsNewStartInEpochNodesCoordinator{
+	eligibleNodesInfo, waitingNodesInfo := args.GenesisNodesConfig.InitialNodesInfo()
+
+	eligibleValidators, err := sharding.NodesInfoToValidators(eligibleNodesInfo)
+	if err != nil {
+		return nil, err
+	}
+
+	waitingValidators, err := sharding.NodesInfoToValidators(waitingNodesInfo)
+	if err != nil {
+		return nil, err
+	}
+
+	consensusGroupCache, err := lrucache.NewCache(50)
```

Comment:
is this enough? Maybe use a const?

Reply:
this nodescoordinator is used only to create the setup. not to compute validators group.

Reply:
added const.